pages/studentPortal/quizcomponent/LangQuizRoute.tsx

```diff
@@ -11,91 +11,177 @@ export enum Stage {
   BLUEPRINT,
   RESULTS,
 }
+
+type QuizSelection = {
+  name: string;
+  result?: string;
+  weight?: number;
+};
+
 const LangQuiz = () => {
```

Comment:
can you change Lang to something more descriptive like FirstProgrammingLanguageQuiz

Reply:
Sounds good! Changes have been implemented.